setup.py

```diff
@@ -14,7 +14,7 @@
     long_description_content_type="text/markdown",
     url="https://github.com/MrDogeBro/sphinx_rtd_dark_mode",
     download_url=f"https://github.com/MrDogeBro/sphinx_rtd_dark_mode/archive/v{sphinx_rtd_dark_mode.__version__}.tar.gz",
-    packages=setuptools.find_packages(),
+    packages=setuptools.find_packages(exclude=["tests*"]),
```

Comment:
```suggestion
    packages=setuptools.find_packages(exclude=["tests"]),
```

No need for the wildcard.